### DISCUSSION TITLE
Running the beta version and updating

### DISCUSSION THREAD

**Discussion by nealhamiltonjr** [mealie-recipes/mealie#1343] [General]
I found out how to pull the beta version via. 

https://nightly.mealie.io/documentation/getting-started/installation/sqlite/

Question...when updating do we have to change the yml file from beta1 to beta2 and 3 etc. or will there be a latest tag?

And, while I don't need email setting right now I'm curious if this can be set latter without issue; Also can these setting be put into the gui so you can configure these settings there instead of a config file? 

This recipe software is awesome..it's filling a gap that's been present for a very long time.

**Comment by hay-kot** [comment]
> Question...when updating do we have to change the yml file from beta1 to beta2 and 3 etc. or will there be a latest tag?

You will have to update that tag to the latest beta manually. Until we hit a "stable" release there won't be a latest tag. If you want to do YOLO updates you can use the nightly tag. I don't plan to push broken code to nightly so you should be able to use it just fine, but there's no promises...

> And, while I don't need email setting right now I'm curious if this can be set latter without issue; Also can these setting be put into the gui so you can configure these settings there instead of a config file?

You can set the Email config anytime, they're read at runtime and not saved. I don't plan on adding GUI configuration for this. I only deploy with Ansible so having a GUI would just hinder deployment for me.

**Comment by michael-genson** [comment]
FYI there is a latest tag now
